src/kernel/arch/x86/paging.zig

```diff
@@ -142,8 +144,14 @@ pub fn init(mem_profile: *const MemProfile, allocator: *std.mem.Allocator) void
 
     mapDir(kernel_directory, p_start, p_end, v_start, v_end, allocator) catch unreachable;
     const dir_physaddr = @ptrToInt(kernel_directory) - @ptrToInt(&KERNEL_ADDR_OFFSET);
-    asm volatile ("mov %[addr], %%cr3" :: [addr] "{eax}" (dir_physaddr));
-    isr.registerIsr(14, pageFault) catch unreachable;
+    asm volatile ("mov %[addr], %%cr3"
+        :
+        : [addr] "{eax}" (dir_physaddr)
+    );
+    isr.registerIsr(14, if (options.rt_test) testPageFault else pageFault) catch unreachable; //else pageFault) catch unreachable;
+    log.logInfo("Done\n");
+
+    if (options.rt_test) runtimeTests(v_end);
 }
 
 test "isAligned" {
```

Comment:
Could the tests be arrange in the same order as the functions.
Also, could you move these tests to a unit test file following my unit testing guide if possible. If not possible, nvm :)

Reply:
You wont be able to move the unit tests to a separate file bc the functions are private. Ignore that comment.